### PR TITLE
Default to not attching AmazonSSMManagedInstanceCore to instances

### PR DESCRIPTION
**Pull request by HenryNguyen5**
Disables AmazonSSMManagedInstanceCore policy attachment to runner instances unless the 
 `debug_instances` variable is set.

Related issue:
https://github.com/philips-labs/terraform-aws-github-runner/issues/114

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ No requirements.
 | vpc\_id | The VPC for security groups of the action runners. | `string` | n/a | yes |
 | webhook\_lambda\_timeout | Time out of the webhook lambda in seconds. | `number` | `10` | no |
 | webhook\_lambda\_zip | File location of the webhook lambda zip file. | `string` | `null` | no |
-
+| debug\_instances | Enable to allow access the runner instances for debugging purposes via SSM. Note that this adds additional permissions to the runner instances. | `bool` | `false` | no |
```
Comment (npalm) [inline README.md:320]: I would prefer a variable like `enable_ssm_on_runners`. Please could you rename?

**Review by npalm — CHANGES_REQUESTED** [review]

**Review by npalm — CHANGES_REQUESTED** [review]
@HenryNguyen5 please check the build, terraform format is failing. The repo has configured https://pre-commit.com/ run terraform format on commint and update the variables in the readme.

**Comment by HenryNguyen5** [comment]
@npalm should be good now!

**Comment by HenryNguyen5** [comment]
Rebased

**Comment by HenryNguyen5** [comment]
@npalm anything I should do to push this along?

**Review by npalm — APPROVED** [review]

**Comment by npalm** [comment]
> @npalm anything I should do to push this along?

Thank, looks all good. We will merge asap

**Pull request merged**